Diagnostics/HealthChecker/Helpers/Class.ps1

```diff
@@ -43,6 +43,8 @@ using System.Collections;
             public Hashtable ApplicationConfigFileStatus = new Hashtable();
             public object DependentServices; // store the results for the dependent services of Exchange.
             public object IISConfigurationSettings; // store the results for the IIS Settings test
+            public object applicationHostConfig; //Stores the applicationHost.config (IIS configuration)
+            public object IISModulesInformation; //Stores the information about modules which are loaded by IIS
```

Comment:
Let's change this to be a single attribute. 

`IISSettings`

Make a function that is `Get-ServerIisSettings` that will return an object that contains this information.

Reply:
Added the 3 cmdlets into a simple `Get-ExchangeServerIISSettings` wrapper function which returns a custom object with `IISConfigurationSettings`, `applicationHostConfig` and `IISModulesInformation`